test/connectivity/modules/NetworkController/SatelliteController.cs

```diff
@@ -1,30 +1,34 @@
 // Copyright (c) Microsoft. All rights reserved.
 namespace NetworkController
 {
+    using System.Runtime.InteropServices;
     using System.Threading;
     using System.Threading.Tasks;
     using Microsoft.Azure.Devices.Edge.ModuleUtil.NetworkController;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
 
-    // TODO: implement satellite
     class SatelliteController : INetworkController
     {
-        string name;
+        static readonly ILogger Log = Logger.Factory.CreateLogger<SatelliteController>();
+        readonly INetworkController underlyingController;
 
-        public SatelliteController(string name)
+        public SatelliteController(string networkInterfaceName, string iotHubHostname, NetworkProfile.ProfileSetting settings)
```

Comment:
nit:internal for all methods for test since it is not used from outside.

Reply:
the class is internal